package.json

```diff
@@ -10,18 +10,17 @@
     "lint": "eslint .",
     "lint:fix": "eslint --fix .",
     "format": "prettier --write '**/*.{js,jsx,json,ts,tsx,yml,md,html,scss,css}'",
-    "format:check": "prettier --check '**/*.{js,jsx,json,ts,tsx,yml,md,html,scss,css}'",
-    "cypress:open": "cypress open",
-    "cypress:watch": "cypress open --config watchForFileChanges=true",
-    "cypress:run": "cypress run",
-    "cypress:run:firefox": "cypress run --browser firefox"
+    "format:check": "prettier --check '**/*.{js,jsx,json,ts,tsx,yml,md,html,scss,css}'"
   },
   "husky": {
     "hooks": {
-      "pre-commit": "lint-staged"
+      "pre-commit": "lint-staged",
+      "pre-push": "git diff HEAD --quiet && npm run test -- --watchAll=false"
```

Comment:
`git diff HEAD --quiet` ensures that local un-committed files do not result in unit tests passing, which then fail in CI with only the committed files.  Trying to push with locally un-committed files will error.  (See https://faun.pub/why-using-pre-push-git-hooks-with-husky-is-not-always-a-good-idea-6233b8afcf83)

Reply:
@zackads @Ninamma 

Sorry to come late to the party on this. I'd find this personally quite annoying and would prefer to have a separate terminal open with `npm run test:watch` running. 

Running the tests on every push could also become frustrating when the test suit grows and takes > 5 seconds to run